### ISSUE TITLE
20180118

### ISSUE THREAD

**Issue by HurleyJames** [HurleyJames/IELTS_Writing#2]
Lectures were used as a way of teaching large numbers of students, but now with the development of technology for education, many people think there is no justification for attending lectures. To what extent do you agree or disagree?

Some people believe that there is no reason to attend traditional lectures since modern educational technology is used currently. However, I cannot agree with this opinion.
Admittedly, modern technology allows individuals to study flexibly. If students cannot attend a lecture punctually, they have to ask others for class notes, but class notes may not help learners fully understand all content given by teachers. However, if students choose to use modern educational technology, such as taking online courses, they can see how teachers explain some difficult questions on the screen at any time when they are free. Nevertheless, admitting the convenience of modern educational technology is not assuming that lectures can be abandoned.
To begin with, attending lectures is an opportunity for students to enlarge their social network. Many students prefer to study in dormitory rooms, but such a way of learning will make learners isolated from others. If students decide to attend lectures, they will meet their classmates, as well as some students from other majors. Therefore, traditional lectures actually allow students to improve their practical skills because they need to communicate face-to-face.
In addition, going to a lecture, students may receive more knowledge. When people take virtual classes online, most class sessions are recorded in advance, and teachers just teach knowledge calmly in front of a camera. However, when giving a lecture, teachers will respond to questions of audiences, and some new knowledge maybe explained to students. In other words, lectures are dynamic, and the interaction between audiences and lectures will bring benefits to audiences, while online courses are fixed. 
To sum up, considering that students are likely to gain more knowledge and exercise their interpersonal skills through attending lectures, traditional lectures still have its advantages compared with modern technology.
